### ISSUE TITLE
Tsv format for library

### ISSUE THREAD

**Issue by Giu-F** [vdemichev/DiaNN#903]
Hi, I generated a library from a fasta file in DIANN and got it in the predicted.speclib format. I need to convert it to .tsv. 
I converted it to .tsv by loading the library, unchecking the two boxes in the “Precursor Generation” tab, then clicking “Run”.
Then I tried to load the .tsv library in DIANN again but it does give this error: ERROR: C:\diann\src\diann.cpp: 19135: Cannot load spectral library. 
Thanks in advance

**Comment by vdemichev** [comment]
Was the .tsv file created? How do all logs look like?

**Comment by Giu-F** [comment]
Yes, the tsv was created in Diann from the predicted.speclib library. This is the log:
DIA-NN 1.8.1 (Data-Independent Acquisition by Neural Networks)
Compiled on Apr 14 2022 15:31:19
Current date and time: Mon Jan  8 11:38:53 2024
CPU: GenuineIntel Intel(R) Xeon(R) CPU E5-2620 v2 @ 2.10GHz
SIMD instructions: AVX SSE4.1 SSE4.2 
Logical CPU cores: 24
diann.exe --lib  --threads 12 --verbose 1 --out F:\GF\report.tsv --qvalue 0.01 --matrices --out-lib F:\GF\lib-400-500.tsv --gen-spec-lib --predictor --fasta \\unicph.domain\groupdir\SUN-CPR-proteomics_jvo\GIULIA\0. CONTAMINANTS\Contaminants Cell Culture.fasta --fasta \\unicph.domain\groupdir\SUN-CPR-proteomics_jvo\Fasta\UP_2023_03\Human\uniprot_9606_reviewed.fasta --fasta-search --min-fr-mz 200 --max-fr-mz 1800 --met-excision --cut K*,R* --missed-cleavages 1 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 400 --max-pr-mz 500 --min-pr-charge 2 --max-pr-charge 4 --unimod4 --var-mods 3 --var-mod UniMod:35,15.994915,M --var-mod UniMod:21,79.966331,STY --monitor-mod UniMod:21 --reanalyse --relaxed-prot-inf --smart-profiling --pg-level 1 --peak-center --no-ifs-removal 

Thread number set to 12
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
A spectral library will be generated
Deep learning will be used to generate a new in silico spectral library from peptides provided
Library-free search enabled
Min fragment m/z set to 200
Max fragment m/z set to 1800
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 1
Min peptide length set to 7
Max peptide length set to 30
Min precursor m/z set to 400
Max precursor m/z set to 500
Min precursor charge set to 2
Max precursor charge set to 4
Cysteine carbamidomethylation enabled as a fixed modification
Maximum number of variable modifications set to 3
Modification UniMod:35 with mass delta 15.9949 at M will be considered as variable
Modification UniMod:21 with mass delta 79.9663 at STY will be considered as variable
A spectral library will be created from the DIA runs and used to reanalyse them; .quant files will only be saved to disk during the first step
Highly heuristic protein grouping will be used, to reduce the number of protein groups obtained; this mode is recommended for benchmarking protein ID numbers; use with caution for anything else
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Implicit protein grouping: protein names; this determines which peptides are considered 'proteotypic' and thus affects protein FDR calculation
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
Exclusion of fragments shared between heavy and light peptides from quantification is not supported in FASTA digest mode - disabled; to enable, generate an in silico predicted spectral library and analyse with this library
The following variable modifications will be scored: UniMod:21 
WARNING: MBR turned off, two or more raw files are required

0 files will be processed
[0:00] Loading FASTA \\unicph.domain\groupdir\SUN-CPR-proteomics_jvo\GIULIA\0. CONTAMINANTS\Contaminants Cell Culture.fasta
[0:00] Loading FASTA \\unicph.domain\groupdir\SUN-CPR-proteomics_jvo\Fasta\UP_2023_03\Human\uniprot_9606_reviewed.fasta
[0:29] Processing FASTA
[3:06] Assembling elution groups
[3:20] 4069884 precursors generated
[3:20] Gene names missing for some isoforms
[3:20] Library contains 20581 proteins, and 20168 genes
[3:22] [3:33] [30:34] [39:51] [40:01] [40:05] Saving the library to F:\GF\lib-400-500.predicted.speclib
[40:23] Initialising library


Finished

This must be wrong because after I upload it, I get this log:
[0:00] Loading spectral library N:\SUN-CPR-proteomics_jvo\GIULIA\6. SPECTRAL_LIBRARIES\Phos\Phos_libraries\lib-phos-400-500.log.txt
WARNING: cannot find column  ModifiedPeptide LabeledSequence FullUniModPeptideName tg_moleculeId "FullUniModPeptideName" modification_sequence 
ERROR: C:\diann\src\diann.cpp: 19135: Cannot load spectral library

DIA-NN exited
